news/move-group-std-top.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* no news added: move group standards to the top document level instead of placing it under FAQ.
```

Comment:
I htink this could actually use a news.  It is an interesting development for users that could be advertised.

Reply:
done